compose/reference/envvars.md

```diff
@@ -4,139 +4,165 @@ keywords: fig, composition, compose, docker, orchestration, cli, reference
+Set this version value in order to match the server version.
 
-Setting this variable is intended as a workaround for situations where you need
-to run temporarily with a mismatch between the client and server version. For
-example, if you can upgrade the client but need to wait to upgrade the server.
 
-Running with this variable set and a known mismatch does prevent some Docker
-features from working properly. The exact features that fail would depend on the
-Docker client and server versions. For this reason, running with this variable
-set is only intended as a workaround and it is not officially supported.
 
-If you run into problems running with this set, resolve the mismatch through
-upgrade and remove this setting to see if your problems resolve before notifying
-support.
+**Important**
+
+> **This setting is only intended as a workaround and it is not officially supported.**
+>
+> Set and use this variable only as a last resort workaround when you need
+> to *temporarily* run Docker with a mismatch between the client and server version. 
+> For example, when you can upgrade the client but need to wait to upgrade the server.
+>
+> This mismatch between server and client prevents some features from working properly. 
+> What features fail when using this configuration depends on the
+> Docker client and server versions in question. 
+>
+> It is highly recommended that you upgrade client and server and remove this setting as 
+> as soon as possible. Also, perform these actions to check if any problems you might 
+> be having are resolved before notifying support.
 
 ## DOCKER\_HOST
 
-Sets the URL of the `docker` daemon. As with the Docker client, defaults to `unix:///var/run/docker.sock`.
+Sets the URL of the Docker daemon. 
+* **Defaults to:** `unix:///var/run/docker.sock`(same as with the Docker client).
 
 ## DOCKER\_TLS\_VERIFY
 
 When set to anything other than an empty string, enables TLS communication with
-the `docker` daemon.
+the Docker daemon.
 
 ## DOCKER\_CERT\_PATH
 
-Configures the path to the `ca.pem`, `cert.pem`, and `key.pem` files used for TLS verification. Defaults to `~/.docker`.
+Configures the path to the `ca.pem`, `cert.pem`, and `key.pem` files used for TLS verification.
+
+* **Defaults to:** `~/.docker`.
 
 ## COMPOSE\_HTTP\_TIMEOUT
 
-Configures the time (in seconds) a request to the Docker daemon is allowed to hang before Compose considers
-it failed. Defaults to 60 seconds.
+Defines the timeout period (in seconds) for requests to the Docker daemon after which Compose considers the requests have failed.
+
+* **Defaults to:** 60 seconds.
 
 ## COMPOSE\_TLS\_VERSION
 
-Configure which TLS version is used for TLS communication with the `docker`
-daemon. Defaults to `TLSv1`.
-Supported values are: `TLSv1`, `TLSv1_1`, `TLSv1_2`.
+Defines which TLS version is used for TLS communication with the Docker daemon. 
+
+* **Supported values:** `TLSv1`, `TLSv1_1`, `TLSv1_2`.
+* **Defaults to:** `TLSv1`.
```

Comment:
Deprecated

Reply:
Added a deprecation notice. Assuming it is better to remove them all in a posterior PR.